src/components/connect.js

```diff
@@ -70,30 +198,30 @@ export default function connect(mapPropsToRequestsToProps, options = {}) {
 
   function assignDefaults(mapping, parent) {
     return Object.assign(
+      { comparison: defaults.comparison },
       parent ? {
```

Comment:
note to self: re-think this whole `parent.comparison` thing. there's probably a better way. (future thing, not this PR)